form/root_storage/root_tbranch_read_container.cpp

```
@@ -31,8 +40,47 @@ void ROOT_TBranch_Read_ContainerImp::setFile(std::shared_ptr<IStorage_File> file
   return;
 }
 
+void ROOT_TBranch_Read_ContainerImp::prime(std::type_info const& type)
```

Cppcheck (2.21.0)

[style] The function 'prime' is never used.

(unusedFunction)